plugins/techdocs/src/search/components/TechDocsSearch.tsx

```diff
@@ -62,15 +58,28 @@ type TechDocsDoc = {
 type TechDocsSearchResult = {
   type: string;
   document: TechDocsDoc;
+  highlight?: ResultHighlight;
+};
+
+const isTechDocsSearchResult = (
+  option: any,
+): option is TechDocsSearchResult => {
+  return option?.document;
+};
+
+const withContext = (Component: ComponentType<TechDocsSearchProps>) => {
```

Comment:
I can't see the component itself was wrapped with the context before, but definitely consumed it. Was that happening somewhere else?

Reply:
+1 -- maybe we can avoid double-wrapping.

Reply:
I did not see that it was already wrapped [here](https://github.com/backstage/backstage/blob/master/plugins/techdocs/src/search/components/TechDocsSearch.tsx#L196), so I will remove it, good catch!

Reply:
Done!